src/components/InsureeMasterPanel.js

```diff
@@ -227,6 +232,47 @@ class InsureeMasterPanel extends FormPanel {
                       onChange={(v) => this.updateAttribute("passport", !!v ? v : null)}
                     />
                   </Grid>
+                  <Grid item xs={3} className={classes.item}>
+                    <PublishedComponent
+                      pubRef="insuree.InsureeStatusPicker"
+                      label="Insuree.status"
+                      value={edited?.status}
+                      module="insuree"
+                      readOnly={readOnly}
+                      onChange={(v) => {
+                        this.updateAttributes({ "status": v, "statusReason": null });
+                      }}
```

Comment:
No need to wrap it in ' { } '.
`
                      onChange={(v) => this.updateAttributes({ "status": v, "statusReason": null })}
`